### PR TITLE
6555 link to Download instructions page & Windows executable

### PR DESCRIPTION
**Pull request by snoek09**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #6555 |

I decided to not copy any instructions as mentioned on https://getcomposer.org/download/

> WARNING: Please do not redistribute the install code. It will change with every version of the installer. Instead, please link to this page.

### CODE REVIEW

File: cookbook/composer.rst [hunk 1]
```diff
@@ -14,28 +14,18 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+Run the installer following the Download page instructions from https://getcomposer.org/download.
+
+After this you can run this command to move composer.phar to a directory that is in your path:
```
Comment (xabbuh) [inline cookbook/composer.rst]: the filename should be enclosed by literals (double backticks)

File: cookbook/composer.rst [hunk 2]
```diff
@@ -14,28 +14,18 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+Run the installer following the Download page instructions from https://getcomposer.org/download.
```
Comment (xabbuh) [inline cookbook/composer.rst]: We should keep a real link to this resource imo.

File: cookbook/composer.rst [hunk 3]
```diff
@@ -14,28 +14,18 @@ following sections.
...
-    .. code-block:: bash
-
-        $ php installer
-        $ sudo mv composer.phar /usr/local/bin/composer
-
 Install Composer on Windows
 ---------------------------
 
-Download the installer from `getcomposer.org/download`_, execute it and follow
+Download the installer from https://getcomposer.org/Composer-Setup.exe, execute it and follow
```
Comment (xabbuh) [inline cookbook/composer.rst]: I would change to something like "Download `Composer-Setup.exe`_, execute [...]" and then add the full URL reference at the bottom to have a link that looks nice.

File: cookbook/composer.rst (hunk 2 above)
Comment (WouterJ) [inline cookbook/composer.rst]: What about using steps here and using nice link names?

```rst
#. Run the installer as described in `the official Composer documentation`_.
#. Execute the following command to move the `composer.phar` to a directory
   that is in your path:

   .. code-block:: bash

       ...
```

File: cookbook/composer.rst (hunk 3 above)
Comment (WouterJ) [inline cookbook/composer.rst]: Let's use a link name here as well (``Download the installer from `getcomposer.org`_, execute it and follow the instructions.``

Then at the bottom of the page, add `.. _getcomposer.org: https://getcomposer.org/Composer-Setup.exe`

**Comment by WouterJ** [comment]
This is a lot more future-proof than the previous article. Thanks @snoek09!

File: cookbook/composer.rst [hunk 4]
```diff
@@ -14,35 +14,24 @@ following sections.
...
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+#. Run the installer as described in `the official Composer documentation`_.
+#. Execute the following command to move the ``composer.phar`` to a directory that is in your path:
 
 .. code-block:: bash
 
-    $ curl -sS https://getcomposer.org/installer | php
     $ sudo mv composer.phar /usr/local/bin/composer
```
Comment (WouterJ) [inline cookbook/composer.rst]: please indent this block with 3 spaces (this way, it's part of the second item of the list)

File: cookbook/composer.rst [hunk 5]
```diff
@@ -14,35 +14,24 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+#. Run the installer as described in `the official Composer documentation`_.
```
Comment (WouterJ) [inline cookbook/composer.rst]: our standards indicate that every list item should end with a semi-colon (except from the last item).

File: cookbook/composer.rst [hunk 6]
```diff
@@ -14,35 +14,24 @@ following sections.
...
-    ``installer`` file manually at https://getcomposer.org/installer and
-    then run:
+#. Run the installer as described in `the official Composer documentation`_;
+#. Execute the following command to move the ``composer.phar`` to a directory that is in your path:
 
     .. code-block:: bash
-
-        $ php installer
+    
         $ sudo mv composer.phar /usr/local/bin/composer
```
Comment (xabbuh) [inline cookbook/composer.rst]: The whole code block must be indented by one space less to not be rendered as a quote.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @snoek09.

**Pull request closed without merging**